src/notification/NotificationController.ts

```diff
@@ -0,0 +1,102 @@
+import BaseController, { BaseConfig, BaseState } from '../BaseController';
+
+interface viewedNotification {
+  [id: number]: boolean;
+}
+
+interface Notification {
+  id: number;
+  date: string;
+  image?: string;
```

Comment:
We could probably get rid of this property too, since nothing in this controller uses this :thinking: 
Not a blocker though I suppose! We can always remove it later.